config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-presubmits.yaml

```diff
@@ -0,0 +1,61 @@
+presubmits:
+  kubernetes-sigs/cluster-api-provider-gcp:
+  - name: pull-cluster-api-provider-gcp-build
+    agent: kubernetes
+    always_run: true
+    context: pull-cluster-api-provider-gcp-build
+    rerun_command: "/test pull-cluster-api-provider-gcp-build"
+    trigger: "/test( all| pull-cluster-api-provider-gcp-build)"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+        args:	
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+  - name: pull-cluster-api-provider-gcp-make
+    agent: kubernetes
+    always_run: true
+    context: pull-cluster-api-provider-gcp-make
+    rerun_command: "/test pull-cluster-api-provider-gcp-make"
+    trigger: "/test( all| pull-cluster-api-provider-gcp-make)"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - args:
+        - "--repo=sigs.k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=90"
+        - "--scenario=execute"
+        - "--"
+        - "make"
+        - "images"
+        env:
+        - name: DOCKER_IN_DOCKER_ENABLED
+          value: "true"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20180716-9145034c9-1.10
+        resources:
+          requests:
+            memory: "6Gi"
+  - name: pull-cluster-api-provider-gcp-test
+    agent: kubernetes
+    always_run: true
+    context: pull-cluster-api-provider-gcp-test
+    rerun_command: "/test pull-cluster-api-provider-gcp-test"
+    trigger: "/test( all| pull-cluster-api-provider-gcp-test)"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/cluster-api:v20180720-d52d72975
+        args:
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
```

Comment:
assume you want to execute something here?

Reply:
Yes, I was just thinking that....

Reply:
Ok, now actually executing something. Turns out that I needed the commands that had previously been hidden in `jobs/config.json`.